src/main/webapp/itinerary-overview.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta
+    charset="UTF-8"
+    name="viewport"
+    content="width=device-width, initial-scale=1.0"
+    />
+    <title>Itinerary Overview</title>
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
+    <script
+            src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDlLtx69Y4-65_dCK67ZX3lzKTYpyc5CWI&callback=initMap&libraries=&v=weekly"
+            defer
+    ></script>
+    <link
+            href="https://fonts.googleapis.com/icon?family=Material+Icons"
+            rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles/constant-style.css" />
+    <link rel="stylesheet" href="styles/itinerary-overview.css" />
+    <link
+            rel="stylesheet"
+            href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css"
+    />    <script src="scripts/itinerary-overview.js"></script>
```

Comment:
Load the script at the end of `<body>` so it doesn't prevent the page from loading first